stations.csv

```diff
@@ -70138,7 +70138,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74279;Zöberitz Bahnhofstraße;zoberitz-bahnhofstrasse;;;51.508217;12.033662;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8070959;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74279;
 74280;Weißandt-Gölzau Tanzfleck;weissandt-golzau-tanzfleck;;;51.671192;12.068135;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8071669;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74280;
 74281;Rostock-Marienehe An der Jägerbäk;rostock-marienehe-an-der-jagerbak;;;54.110584;12.080325;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8071759;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74281;
-74282;Rostock-Marienehe Sporthalle;rostock-marienehe-sporthalle;;;54.111465;12.081457;;DE;Europe/Berlin;f;f;f;f;f;f;;;f;;f;8071758;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74282;
+74282;Rostock-Marienehe Sporthalle;rostock-marienehe-sporthalle;;;54.111465;12.081457;;DE;Europe/Berlin;f;f;f;f;f;f;;15461;f;;f;8071758;t;;f;;f;;f;;f;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74282;
```

Comment:
You've changed the `sncf_tvs_id` column here instead of the `parent_station_id` column

Reply:
Thank you. I took one look at the test errors and thought, "that can wait until the new year". Suspect this is not the only problem.